test/data/electrical.json

```diff
@@ -37,12 +35,12 @@
               }
             },
             "current": {
-              "value": 0.1,
+              "value": 124.2,
```

Comment:
Just showcasing different capabilities between the two demo batteries - one large, one smaller.